data/kernels_and_extensions.yaml

```diff
@@ -0,0 +1,22 @@
+Links:
+  ansible: https://github.com/ansible/ansible-jupyter-kernel
+  bash: https://github.com/takluyver/bash_kernel
+  c++: https://github.com/jupyter-xeus/xeus-cling
+  cling: https://github.com/jupyter-xeus/xeus-cling
+  golang: https://github.com/gopherdata/gophernotes
+  gurobi: https://www.gurobi.com/documentation/
+  gnuplot: https://github.com/has2k1/gnuplot_kernel
+  javaScript: https://github.com/n-riesco/ijavascript
+  julia: https://github.com/JuliaLang/IJulia.jl
+  matlab 2020b: https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/system-requirements-release-2020b-linux.pdf
+  matlab 2022b: https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/system-requirements-release-2022b-windows.pdf
```

Comment:
These are no links to Jupyter kernels. And we have the reference in line 22. So you can remove the matlab links here.